webapp/src/DOMJudgeBundle/Resources/views/jury/team_affiliation.html.twig

```diff
@@ -54,14 +54,14 @@
                     <tr>
                         <th>Country</th>
                         <td>
-                            {{ teamAffiliation.country }}
                             {% if teamAffiliation.country is not empty %}
                                 {% set countryFlag = 'images/countries/' ~ teamAffiliation.country ~ '.png' %}
                                 {% if countryFlag | assetExists %}
                                     <img src="{{ asset(countryFlag) }}" alt="{{ teamAffiliation.country }}"
                                          title="{{ teamAffiliation.country }}" class="countryflag"/>
```

Comment:
Probably the tooltip is not really necessary in this context, since the full name is printed immediately after.